packages/client/verifiable-credentials/src/presentation/getCredential.ts

```diff
@@ -38,3 +66,28 @@ export async function getCredentialFromId(
         return null;
     }
 }
+
+export async function getCredential(
+    collection: CredentialsCollection,
+    tokenId: string,
+    environment: string
+): Promise<VerifiableCredentialType | null> {
+    const metadata = collection.metadata;
+    if (!isVerifiableCredentialContractMetadata(metadata)) {
+        throw new Error(`The collection provided is not a verifiable credential collection`);
+    }
+    const retrievalEndpoint = metadata.credentialMetadata.credentialsEndpoint;
+    // shall we route ipfs requests through the crossmint API?
```

Comment:
Once we decouple the crossmint part, it makes sense to have two different flows and when using crossmint probably we deliver a better ux proxying the request. For now maybe is better to leave it like this